erpnext/accounts/doctype/subscription/subscription.js

```diff
@@ -1,75 +1,78 @@
-// Copyright (c) 2017, Frappe Technologies Pvt. Ltd. and contributors
+// Copyright (c) 2018, Frappe Technologies Pvt. Ltd. and contributors
 // For license information, please see license.txt
 
 frappe.ui.form.on('Subscription', {
-	setup: function(frm) {
-		frm.fields_dict['reference_doctype'].get_query = function(doc) {
-			return {
-				query: "erpnext.accounts.doctype.subscription.subscription.subscription_doctype_query"
-			};
-		};
-
-		frm.fields_dict['reference_document'].get_query = function() {
-			return {
-				filters: {
-					"docstatus": 1,
-					"subscription": ''
-				}
-			};
-		};
-
-		frm.fields_dict['print_format'].get_query = function() {
-			return {
-				filters: {
-					"doc_type": frm.doc.reference_doctype
-				}
-			};
-		};
-	},
-
 	refresh: function(frm) {
-		if(frm.doc.docstatus == 1) {
-			let label = __('View {0}', [frm.doc.reference_doctype]);
-			frm.add_custom_button(__(label),
-				function() {
-					frappe.route_options = {
-						"subscription": frm.doc.name,
-					};
-					frappe.set_route("List", frm.doc.reference_doctype);
-				}
-			);
+		if(!frm.is_new()){
+			if(frm.doc.status !== 'Canceled'){
```

Comment:
Check spelling "Cancelled"

Reply:
@nabinhait I actually mistakenly used the American spelling (VS Code spell checker caused it)... and it's not just here. Should I leave it as 'canceled' since it's technically a correct spelling (American)?